field.go

```diff
@@ -436,95 +455,12 @@ func newDefinition(schema any) Schema {
 		if err != nil {
 			panic(err)
 		}
-		if b == true {
+		if b {
 			newDef.Required = append(newDef.Required, fieldName)
 		}
 		newDef.Properties[fieldName] = &nd
 	}
-	AddDefinition(st.Name(), newDef)
+	Definitions[st.Name()] = &newDef
 	newSchema.Ref = "#/definitions/" + st.Name()
 	return *newSchema
 }
-
-func handleInputSchema(pa *[]Parameter, rb *RequestBodyOrReference, s any) error { // should this return an error or should it panic?
```

Comment:
😭😭

does this code still work? i feel like new bugs just happened cause of this removal.